templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,181 @@
+---
+title: "[Institution-landing-page]"
+layout: no-container
+language: "en"
+altLangPage: "institution-landing-fr.html"
+breadcrumb: [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+	{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+]
+dateModified: "2021-06-03"
+share: true
+---
+<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
```

Comment:
Add the following include before this line:
```suggestion
{%- include variable-core.liquid -%}
<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
```
This allows includes to have access to the i18nText-lang variable.